crates/indexer/src/models/v2_objects.rs

```diff
@@ -157,4 +152,44 @@ impl Object {
             Ok(None)
         }
     }
+
+    /// This is actually not great because object owner can change. The best we can do now though
+    fn get_object_owner(
+        conn: &mut PgPoolConnection,
+        object_address: &str,
+    ) -> anyhow::Result<CurrentObject> {
+        let mut retried = 0;
+        while retried < QUERY_RETRIES {
+            retried += 1;
+            match CurrentObjectQuery::get_by_address(object_address, conn) {
+                Ok(res) => {
+                    return Ok(CurrentObject {
```

Comment:
I thought we also planned to do this for table items?

Reply:
Did you read the summary lol. We can't do this for table items performantly.